website/src/parsers/markdown/transformers/remark/codeExample.txt

```diff
@@ -0,0 +1,5 @@
+module.exports = function attacher(options) {
+  return function transformer(tree, vfile) {
+    return tree;
```

Comment:
It would be great to show an example that actually changes something (I assume this one doesn't), but that's no reason to block this PR.

Reply:
sure thing, opened https://github.com/fkling/astexplorer/pull/551 with a more complete example